int/tests/conftest.py

```diff
@@ -168,3 +169,55 @@ async def http_endpoint(agent: BaseAgent):
     with suppress(asyncio.CancelledError):
         await server_task
     await agent.cleanup()
+
+
+@pytest.fixture
+async def make_did():
+    """DID factory fixture"""
+
+    def _make_did():
+        did = create_did.asyncio(client=backchannel)
+        return did
```

Comment:
```suggestion
    async def _make_did():
        return await create_did.asyncio(client=backchannel)
```

We'll need the async and awaits in here :slightly_smiling_face: